src/util/required-opts.js

```diff
@@ -0,0 +1,12 @@
+var _ = require('underscore');
+
+module.exports = function checkAndBuildRequiredOpts (actualOpts, requiredOpts, context) {
+  if (requiredOpts === void 0) {
```

Comment:
since `void 0` is equivalent to `undefined` IMO we should use [isUndefined](http://underscorejs.org/#isUndefined)

```javascript 

if (_.isUndefined(requireOpts)){
    throw new Error('Opts are required');
}

Reply:
I'd bet that line will always say `Opts are required but received: undefined`.